README.md

```diff
@@ -230,6 +230,7 @@ Additional lists you might find useful:
 - [Ajax plugin](https://github.com/dereuromark/cakephp-ajax) - A plugin to ease handling AJAX requests.
 - [CakeAdmin plugin](https://github.com/cakemanager/cakephp-cakeadmin) - A non-stable user management plugin with a built-in admin area.
 - [CakeMiddlewares](https://github.com/chrisShick/CakeMiddlewares) - A collection of Cakephp Middlewares. Requires CakePHP version 3.4+.
+- [Comments plugin](https://github.com/Kareylo/CakePHP-Comments) - A fully customizable Comments plugin. Requires CakePHP version 3.3.2+.
```

Comment:
The versions can be removed IMO. thats part of the readme and composer constraint.